### PR TITLE
MSPSDS-863 add link to app after welcome journey

### PR DESCRIPTION
**Pull request by jasiekmiko**
The final solution is a bit of a hack - see MSPSDS-890 for details.
Happy to talk through it!

### CODE REVIEW

File: govuk/login/template.ftl [hunk 1]
```diff
@@ -100,7 +100,7 @@
         <main role="main" class="govuk-main-wrapper" id="main-content">
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-two-thirds">
-                    <h1 class="govuk-heading-xl"><#nested "title"></h1>
+                    <h1 class="govuk-heading-xl"><#nested "header"></h1>
```
Comment (tom-bell-softwire) [inline govuk/login/template.ftl]: Out of interest, what's the difference between `header` and `title`?

File: govuk/login/info.ftl [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
-                <p><a class="button" href="${client.baseUrl}">${msg("backToApplication")?no_esc}</a></p>
-            </#if>
-        </#if>
-    </div>
+                <#--
+                    This is a hack to direct the users to the applications without the ability to supply the link
+                    through the magic link redirect parameter.
+                    TODO MSPSDS-890
+                -->
+                <p><a class="button" href="https://mspsds-prod.london.cloudapps.digital">Market Surveillance & Product Safety</a></p>
```
[the remark this answers is not in the document]
Comment (jasiekmiko) [reply govuk/login/info.ftl]: Yes, that is the problem - in conjunction with the fact that the default client is the account one, which leads to the double-login problem (more details in MSPSDS-890)

File: govuk/login/template.ftl [hunk 3]
```diff
@@ -67,7 +67,7 @@
     <header class="govuk-header" role="banner" data-module="header">
         <div class="govuk-header__container govuk-width-container">
             <div class="govuk-header__logo">
-            <a href="#" class="govuk-header__link govuk-header__link--homepage">
+            <a href="/auth/realms/mspsds/account/applications" class="govuk-header__link govuk-header__link--homepage">
```
[the remark this answers is not in the document]
Comment (jasiekmiko) [reply govuk/login/template.ftl]: I'll check both those options.
Good spot with `opss` though!

File: govuk/login/template.ftl (hunk 1 above)
Comment (jasiekmiko) [reply govuk/login/template.ftl]: They are names of sections provided by the templates that extend this layout. Somehow the layout used to ignore the header and duplicate the title. I believe the intention is for the header to be the h1 element of the page, while title to be part of the page's html title (e.g. displayed in the tab)

File: govuk/login/template.ftl (hunk 3 above)
[the remark this answers is not in the document]
Comment (jasiekmiko) [reply govuk/login/template.ftl]: As discussed, reverted back to `#` for the time being

**Pull request merged**